searchcore/src/tests/proton/attribute/attributeflush_test.cpp

```diff
@@ -479,13 +479,13 @@ Test::requireThatOnlyOneFlusherCanRunAtTheSameTime()
     AttributeManager &am = f._m;
     AttributeVector::SP av = f.addAttribute("a8");
     av->addDocs(10000);
-    av->commit(9,9);
+    av->commit(CommitParam(9, true));
     IFlushTarget::SP ft = am.getFlushable("a8");
     (static_cast<FlushableAttribute *>(ft.get()))->setCleanUpAfterFlush(false);
     vespalib::ThreadStackExecutor exec(16, 64000);
 
     for (size_t i = 10; i < 100; ++i) {
-        av->commit(i, i);
+        av->commit(CommitParam(i));
```

Comment:
This is the only place we use  _forceUpdateStats=false_, as before. Why is that?

Reply:
Removed semantic change